frontend/components/Domain/Recipe/RecipeContextMenu.vue

```diff
@@ -347,6 +370,19 @@ export default defineComponent({
     }
 
     const router = useRouter();
+    const groupRecipeActionsStore = useGroupRecipeActions();
+
+    async function executeRecipeAction(action: GroupRecipeActionOut) {
+      await groupRecipeActionsStore.execute(action, props.recipe);
+
+      switch (action.actionType) {
+        case "post":
+          alert.success(i18n.tc("events.message-sent"));
+          break;
+        default:
+          break;
+      }
```

Comment:
This seems a bit unnecessairy for pretty much just one case. Do you do it this way for future expansion?

If not, why not a simple if statement?

```suggestion
      if (action.actionType === "post") {
        alert.success(i18n.tc("events.message-sent"));
      }
      break;
```

Reply:
Yeah I had this as a switch statement because I expected to have more types, but I agree it's premature now